homeassistant/components/advantage_air/sensor.py

```diff
@@ -0,0 +1,212 @@
+"""Sensor platform for Advantage Air integration."""
+import voluptuous as vol
+
+from homeassistant.const import PERCENTAGE
+from homeassistant.helpers import config_validation as cv, entity_platform
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import ADVANTAGE_AIR_STATE_OPEN, DOMAIN
```

Comment:
DOMAIN should be exclusive to the platform domain itself (sensor)
```suggestion
from .const import ADVANTAGE_AIR_STATE_OPEN, DOMAIN as ADVANTAGE_AIR_DOMAIN
```